pkg/cli/host.go

```diff
@@ -35,7 +35,16 @@ type createGCPHostFlags struct {
 	MinCPUPlatform string
 }
 
-func newHostCommand(cfgFlags *configFlags) *cobra.Command {
+type GCPHostConfig struct {
```

Comment:
could this be moved alongside the more general Config structure? having the config struct/parameters closer to each other makes it more easy to scan through all of them at once.

Reply:
Done